service/src/main/kotlin/xyz/lebkuchenfm/domain/eventstream/Event.kt

```diff
@@ -9,7 +9,7 @@ sealed interface Event {
     ) : Event
 
     data class QueueSongs(
-        val songs: List<Song>,
+        val songs: List<SongWithVideoStream>,
```

Comment:
now domain event is different than EventDTO. I get a point but at first glance it is a bit confusing. Had to check twice if we did or did not introduce breaking change here.